src/main/java/com/extendedclip/deluxemenus/menu/options/MenuItemOptions.java

```diff
@@ -512,8 +491,8 @@ public MenuItemOptionsBuilder bannerMeta(final @NotNull List<Pattern> bannerMeta
             return this;
         }
 
-        public MenuItemOptionsBuilder itemFlags(final @NotNull List<ItemFlag> itemFlags) {
```

Comment:
Small thing, and we can deal with this if we ever have to, but there's no way to remove an item flag.

Reply:
It wasn't before either. What do you suggest?

Reply:
@BlitzOffline would this work?
Two `Set<ItemFlag>`, one for flags that needs to be added and one to be removed. Both sets will be read from `item_flags` and the action will be distinguished by a prefix, something like this:
```yaml
item_flags:
- HIDE_ENCHANTMENTS
- remove:HIDE_ATTRIBUTES
# or ~<flag> (or another symbol)
# ! doesn't work because it is used by yaml to specify the type of the value
- ~HIDE_ATTRIBUTES
```

Reply:
As I've mentioned, this is not something we really need to deal with right now since we don't actually have a use case for this and that is bcz if `<item_flag_option>: <boolean>` is set to false, we just ignore the option.

Also, I don't think we need to overcomplicate things. We can just rename this method to `addItemFlags` and add a new one to `removeItemFlags`. Especially since the item flags are not held in immutable collections anymore.

Reply:
By 'remove' you mean `ItemMeta#removeItemFlags`, right?